_topic_maps/_topic_map_osd.yml

```diff
@@ -186,11 +186,32 @@ Name: Networking
 Dir: networking
 Distros: openshift-dedicated
 Topics:
+- Name: Understanding the Ingress Operator
+  File: ingress-operator
+- Name: Configuring the Ingress Controller endpoint publishing strategy
```

Comment:
This should not be configurable by the customer. We should state that this is SRE managed via OCM+Hive.